src/woo_publications/metadata/management/commands/update_information_category.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from woo_publications.metadata.update_informatie_category import (
+    InformatieCategoryWaardenlijstError,
+    update_informatie_category,
+)
+
+
+class Command(BaseCommand):
+    help = "Used to fetch the gov waardenlijsten data and turn them into a fixture to load the data into the db."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--file-path",
+            action="store",
+            help="The file path to where the fixture file will be created.",
+            default=None,
```

Comment:
I would specify the default value here in the command, and not in the function that implements it. The function itself can then be 100% agnostic about file paths.

Looking at the implementation of `dumpdata`, I think you can also pass `pathlib.Path` instances to `output` instead of converting it to a string first.